CHANGELOG.rst

```diff
@@ -7,6 +7,14 @@ Change Log
 ----------
 
 
+6.7.1
```

Comment:
New functionality is a minor version change. We're at 6.8 so you'll need to be 6.9.0.

Reply:
Got it.